src/core/Akka/Actor/FSM.cs

```diff
@@ -671,7 +671,7 @@ public override string ToString()
     /// <typeparam name="TData">The state data type</typeparam>
     public abstract class FSM<TState, TData> : FSMBase, IListeners, IInternalSupportsTestFSMRef<TState,TData>
     {
-        private readonly ILoggingAdapter _log = Context.GetLogger();
+        private readonly ILoggingAdapter _log = Context.GetLoggerStartup();
```

Comment:
So - does this mean that if logger did not get the remote context on start, then this actor instance will never log with remoting context?
I am not familiar with how this "remoting context" thing impacts the logger output, but just thinking if there is some way to give it a "retry" mechanism so that once remoting is up and running, the loggers could catch it up somehow. 

Well, maybe `GetLoggerStartup` could receive `ILoggingAdapter currentLogger` parameter and see if this logger has remoting context skipped and has to be replaced, or `currentLogger` is good to go... Too complicated?

Of course my thoughts make sense only if having remoting context in logger is important - otherwise, current fix should be everything we need 👌

Reply:
> So - does this mean that if logger did not get the remote context on start, then this actor instance will never log with remoting context?

That's correct - should only happen for a small number of `/system` actors that are launched before the transport is bound (i.e. the ones responsible for managing that transport)

Reply:
> I am not familiar with how this "remoting context" thing impacts the logger output, but just thinking if there is some way to give it a "retry" mechanism so that once remoting is up and running, the loggers could catch it up somehow.

What we do currently is basically this - crash the actor and re-attempt to acquire the transport after it's been bound. Unfortunately when Visual Studio is attached users intercept the error and report it, which has happened in at least 3 separate issues now. This is my attempt to put it to bed for good.

Reply:
Ah, I see - then there is nothing to worry about